### DISCUSSION TITLE
Project Submission Error

### DISCUSSION THREAD

**Discussion by Rakeshgodumala** [github/india#797] [Students]
Hello,
Sir I am trying to deploy my project for the last two weak several times it is showing an Error and changing the Azure App Service to submit my project but, it was unable to submit.

I used Web App it shows Error to deploy a project 
after that I used Static Web App to deploy a project then in GitHub Action it is showing Error in Status (Failure) came.
after that in vs code already install nodejs 16 and npm8
but on the Error page, it is showing installed node js14.18.3
please solve my issue to submit my project

Currently, My Exam is going  and it will be complete on March 12th that's  why unable to complete all self-learning MODULES 
please Extend the Project Submission date  so I can do the Microsoft Future Ready Talent  internship completely
project error video
[https://drive.google.com/file/d/1HSsFbfaOAHGDgIV89eur_ycEeFzC1iQo/view](url)

Please do Help

**Comment by ey-mentor1** [comment]
Hello,
From shared error, it looks like removing package-lock.json from code will work.
Also, can you check, if the path where npm run is executing, is there any package.json in that path.

If possible, can you share git repo, i can try to replicate the issue and then provide the solution.

Else, you can always host static code on azure storage also.

Thank You
